docs/game_sdk/Store.md

```diff
@@ -0,0 +1,454 @@
+
+| name      | value           | description                   |
+| --------- | --------------- | ----------------------------- |
+| sku_types | Discord.SkuType | the type of SKU to filter for |
+
+###### Example
+
+```
+curl https://discordapp.com/api/v6/store/applications/461618159171141643/skus?sku_types=1 \
+-H "Authorization: Bearer <token>" \
+-H "Accept: application/json"
+
+// Returns
+
+{
+  [
+    {
+      "id": "53908232599983680",
+      "type": 1,
+      "dependent_sku_id": null,
+      "application_id": "461618159171141643",
+      "manifest_labels": ["461618159171111111"],
+      "name": "My Awesome Test Game",
+      "access_type": 1,
+      "features": [1, 2, 3],
+      "system_requirements": {},
+      "content_ratings": {},
+      "release_date": "1999-01-01",
+      "legal_notice": {},
+      "price_tier": 1099,
+      "price": {},
+      "premium": false,
+      "locales": ["en-US"],
+      "bundled_skus": null
+    }
+  ]
+}
+```
+
+## Consume SKU % POST /store/applications/{application.id#DOCS_GAME_SDK_SDK_STARTER_GUIDE/get-set-up}/entitlements/{entitlement.id#DOCS_GAME_SDK_STORE/data-models-entitlement-struct}/consume
+
+Marks a given entitlement for the user as consumed, meaning it will no longer be returned in an entitlements check. **Ensure the user was granted whatever items the entitlement was for before consuming it!**
+
+###### Example
+
+```
+curl -X POST https://discordapp.com/api/v6/applications/461618159171141643/entitlements/53908232506183999/consume \
+-H "Authorization: Bearer <token>" \
+-H "Accept: application/json"
+
+// Returns
+
+{
+  "code": 0,
+  "message": "200: OK"
+}
+```
+
+## Get User Entitlements % GET /users/@me/applications/{application.id#DOCS_GAME_SDK_SDK_STARTER_GUIDE/get-set-up}/entitlements
+
+> warn
+> This endpoint falls under the `/users/@me` route schema, which means that it takes the OAuth2 bearer token of a player. This can be retrieved via the Application Manager in the SDK
+
+Returns the entitlements a user has for your application.
+
+###### Example
+
+```
+curl https://discordapp.com/api/v6/users/@me/applications/461618159171141643/entitlements \
+-H "Authorization: Bearer <token>" \
+-H "Accept: application/json"
+
+// Returns
+
+{
+  [
+    {
+      "user_id": "53908232506183680",
+      "sku_id": "53908232599983680",
+      "application_id": "461618159171141643",
+      "id": "53908232506183999",
+      "type": 3
+    }
+  ]
+}
+```
+
+## Checking DLC Entitlements
+
+If your game has DLC, and a user has purchased that DLC, you may want to check what they should have access to when the game launches. DLC entitlements will always be returned in a `FetchEntitlements()` call, so your game can check on each startup whether or not a user should have access to a certain new zone, raid, map, etc. based on their entitlements for DLC.
+
+Entitlements to DLCs are intended to be for SKUs that will download additional data onto a user's hard drive, like an expansion pack for a game.
```

Comment:
While this is an intention, some some devs just have embedded content that unlocks with entitlement. That is a fine way to do things. I need to confirm that there is no issue with using DLC in the "unlock this thing" fashion...there should not be.

Reply:
Apparently a bunch of people use DLC for IAP-like reasons so ima remove that sentence